NEWS.md

```diff
@@ -1,5 +1,9 @@
 # pkgdown (development version)
 
+* All external assets are now directly included in the site by default instead of
```

Comment:
It's not clear to me that it's worth leaving as an option because of https://dev.to/rstacruz/public-cdns-arent-useful-anymore-2b66

Reply:
I have no strong opinion about providing the `template.params.external_assets` config option (I don't use it, i.e. leave it at its default of `false`). I added it to let users easily switch to the old behaviour in case there is a valid reason to want that (but maybe there really isn't).

Let me know if I should refactor the PR to completey remove the option.

Reply:
As far as I know, there's now no valid reason to use a CDN, so I don't think there's any benefit to keeping this as an option, and it would keep pkgdown simpler to only have one code path to test/maintain etc.

Reply:
I've removed the `template.params.external_assets` config option in the latest commit.

Reply:
Another open question: Is it fine to [switch to the minified version of `fuse.js`](https://github.com/r-lib/pkgdown/pull/2249#issuecomment-1337776994)?

Reply:
Yes, for sure.

Reply:
Ok, done in https://github.com/r-lib/pkgdown/pull/2249/commits/810dd6f96192e490ca1a72e0c22dedb0ba98b0a4.